src/index.html

```diff
@@ -4,9 +4,390 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="menu">
+      <div class="menu__container">
+        <a href="#home" class="menu__logo">
+          <img src="./images/logo.png" alt="MOYO logo">
+        </a>
+
+        <nav class="menu__nav nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+              class="nav__link is-active"
+              href="#Apple"
+              >
```

Comment:
please be attentive with code format
```suggestion
              <a
                class="nav__link is-active"
                href="#Apple"
              >
```